art/configs/logger/neptune.yaml

```diff
@@ -0,0 +1,5 @@
+_target_: lightning.pytorch.loggers.NeptuneLogger
+api_key: ${oc.env:NEPTUNE_API_TOKEN} # api key is loaded from environment variable
```

Comment:
Is the key uploaded in GH secrets, or everyone should use his API token? 
If second, should we write you our emails to give us access to the Neptune?